### PR TITLE
Add profile picture, update AboutUs.adoc and ContactUs.adoc

### LINKED ISSUE #43: Update project website: Contact Us page
week 7 task

### LINKED ISSUE #53: Update About Me Page
Add the following:
1. Photo
2. Role and Responsibilities

### PR DESCRIPTION
**Pull request by Exeexe93**
Add profile image for Eng Xuan En and profile page engxuanen.adoc.
Update the roles and responsibles for Eng Xuan En,
Update the link for issue tracker and team mates emails in ContactUs.adoc,

### CODE REVIEW

**Review by JeremyLoh — CHANGES_REQUESTED** [review]
The image should be named via the following format: [githbub_username_in_**lower_case**].png
It is currently engxuanen.png
Your github username is Exeexe93, so it should be **exeexe93.png**.
Remember to change your display picture link in AboutUs.adoc

File: docs/ContactUs.adoc [hunk 1]
```diff
@@ -2,6 +2,11 @@
...
-* *Bug reports, Suggestions* : Post in our https://github.com/se-edu/addressbook-level3/issues[issue tracker] if you noticed bugs or have suggestions on how to improve.
+* *Bug reports, Suggestions* : Post in our https://github.com/AY1920S2-CS2103T-W12-1/main/issues[issue tracker] if you noticed bugs or have suggestions on how to improve.
 * *Contributing* : We welcome pull requests. Follow the process described https://github.com/oss-generic/process[here]
-* *Email us* : You can also reach us at `damith [at] comp.nus.edu.sg`
+* *Email us* : You can also reach us at
+    ** `e0325707 [at]  u.nus.edu`
+    ** `e0318484 [at] u.nus.edu`
+    ** `jeremyloh [at]  u.nus.edu`
+    ** `e0319018 [at]  u.nus.edu`
+    ** `e0318636 [at]  u.nus.edu`
```
Comment (JeremyLoh) [inline docs/ContactUs.adoc]: I think we should add the names of the person for the email at the start. I think its hard to know who has what email.

**Comment by Exeexe93** [comment]
Change the name of the Profile image for Eng Xuan En to exeexe93.png

**Review by khsc96 — APPROVED** [review]
LGTM, commit changed DG docs as well

**Review by JeremyLoh — APPROVED** [review]
LGTM

**Pull request merged**